### PR TITLE
[DEV-3064] Python 3.7

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
~Python 3.5~ -> **Python 3.7**

**Technical details:**
- Removing Python 3.5 config and files
- Updated Docker file and README
- Re-organized Python library files
- Fixed minor issue with load_reference_data.py
- Added flag for matview_runner.py

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated (N/A)
3. [x] Necessary PR reviewers:
    - [x] Backend
    - [x] Operations
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed (N/A)
7. [x] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-3064](https://federal-spending-transparency.atlassian.net/browse/DEV-3064):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected (N/A)
    - [x] Before / After data comparison (N/A)

**Area for explaining above N/A when needed:**
```
Any code changes were to handle the upgrade, not to alter API behavior
```

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -20,3 +20,10 @@ ES_HOSTNAME=http://usaspending-es:9200
 
 # ES_CLUSTER_DIR should point to a path outside, or it will be mounted w/Docker
 ES_CLUSTER_DIR=../docker_es
+
+# Configuration values for a connection string to a Broker database
+#    Only necessary for some management commands
+BROKER_USER=root
+BROKER_PASSWORD=password
+BROKER_HOST=broker-db
+BROKER_PORT=5432
```
Comment (BrianZito) [inline .env]: Can we specify what environment this is for? Seems scary having plaintext Passwords here unless these are just dev or something.

Comment (tony-sappe) [reply .env]: What should be done to make it less scary? The values are obviously for Broker and aren't pointing to an actual DB

Comment (BrianZito) [reply .env]: Are these placeholder values that a user is supposed to fill in? At first glance I would assume this went to _some_ database.

Comment (BrianZito) [reply .env]: Looking at the rest of this file, it is consistent. It just ticked off some alarms when I assumed it was in code.

**Comment by tony-sappe** [comment]
Adding "Do Not Merge" until operations is ready

**Pull request merged**